components/support/ffi/src/lib.rs

```diff
@@ -573,7 +573,7 @@ impl Default for ByteBuffer {
     #[inline]
     fn default() -> Self {
         Self {
-            len: 0 as i64,
+            len: 0_i64,
```

Comment:
Oh, huh, I haven't seen this syntax before and I'm not sure I like it...but I do like not having to argue about these things, so will just let clippy to its thing 😅

Reply:
Agreed; "You too can write Rust to be just as legible as Perl".  :-)